openid-provider-commands-1_0.md

```diff
@@ -428,7 +428,7 @@ The RP MUST include the state of the Account and any Claims for an Account that
 Identified by the `unauthorize` value in the `command` Claim in a Command Token.
 
 The RP MUST perform the [Unauthorize Functionality](#unauthorize-functionality) on the Account.
-The OP sends this Command when it wants the RP to undo the last OpenID Connect login. The OP may send this when it suspects a previous OpenID Connect ID Token issued by the OP was granted to a malicious actor, or if the user's device was compromised. 
+The OP sends this Command when it wants the RP to undo previous OpenID Connect logins. The OP may send this when it suspects a previous OpenID Connect ID Token issued by the OP was granted to a malicious actor, or if the user's device was compromised. 
```

Comment:
I suggest replacing "undo OpenID Connect logins" by "terminate authenticated sessions".

Instead of "terminating all authenticated sessions" which could very painful you might want to terminate a particular with the help of claim "id_token_hint".

The OpenID Connect Core spec provide a mechanism for this.

https://openid.net/specs/openid-connect-core-1_0.html#:~:text=the%20OpenID%20Provider.-,id_token_hint,-OPTIONAL.%20ID%20Token

I also think this whole section needs more clarity.

Reply:
I believe the intent of this command _is_ to terminate _all_ sessions, not just one. It is meant to be the nuclear option, requiring the user signs back in through the OP before being able to use the RP app at all.